requirements.txt

```diff
@@ -2,3 +2,5 @@ pandas
 httplib2
 google-api-python-client
 oauth2client
+google-cloud
```

Comment:
+1 to using `google-cloud-bigquery`.

Also, the `google-cloud-bigquery` package is still beta, meaning there will likely be breaking changes, so we'd have to pin a very specific version number to maintain stability.